OOStepByStep/Class.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OOStepByStep
+{
+    public class Class
+    {
+        // private Person person;
+        // private Teacher teacher;
+        private List<Student> students = new ();
+        private int classNumber;
+
+        public Class(int classNumber)
+        {
+            this.classNumber = classNumber;
+        }
+
+        public string ClassMember(Person person)
```

Comment:
introduce of teacher or student should belong to related class.
e.g teacher.Introduce(), student.Introduce()